invisible_cities/cities/esmeralda_test.py

```diff
@@ -308,4 +308,4 @@ def test_esmeralda_exact_result_all_events(ICDATADIR, KrMC_hdst_filename, correc
             for table in tables:
                 got      = getattr(     output_file.root, table)
                 expected = getattr(true_output_file.root, table)
-                assert_tables_equality(got, expected)
+                assert_tables_equality(got, expected, 1 if table in ("RECO/Events", "PAOLINA/Events") else 0)
```

Comment:
Rather than changing the test file here maybe we can temporarily use dataframes comparison excluding the Ep column since #675 will be changing the test file again ?

Reply:
Can we do that for #675 only? Or do you thing that dirty fix is too unreadable?

Reply:
I think the fix would be more readable if you use dataframe comparison for both esmeralda and penthesilea test, rather than changing testing_utils
but beside that I am nor sure its worth it to change esmeralda files since they will be changed again in #675. Though since it is already in git lfs I guess it doesnt matter